### PR TITLE
[13.0][FIX] account_banking_mandate: Set Force company

### PR DESCRIPTION
**Pull request by etobella**
Missing `force_company`

### CODE REVIEW

**Comment by pedrobaeza** [comment]
In v13 this is done other way I think: https://www.odoo.com/documentation/13.0/howtos/company.html

File: account_banking_mandate/models/account_move.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Marçal Isern <marsal.isern@qubiq.es>
+ # Copyright 2020 Marçal Isern <marsal.isern@qubiq.es>
```
Comment (StefanRijnhart) [inline account_banking_mandate/models/account_move.py]: unintended whitespace?

**Comment by StefanRijnhart** [comment]
> In v13 this is done other way I think: https://www.odoo.com/documentation/13.0/howtos/company.html

Can you clarify @pedrobaeza ? That page still mentions force_company, and so does the code https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/ir_property.py#L185

**Comment by pedrobaeza** [comment]
Yeah, my fault. The change is from saas-13.2/v14, that `with_company` is introduced.

**Comment by pedrobaeza** [comment]
FYI: https://www.odoo.com/documentation/master/howtos/company.html

**Comment by pedrobaeza** [comment]
Any way, CI is red. I plan to work on moving these modules to computed writable fields during Spanish OCA Days, so if you want to wait (and if you participate in the CS), we can make directly the switch for both.

**Comment by StefanRijnhart** [comment]
Thanks for the clarification. I assume that if pre-commit nags are fixed quickly this can be merged in the meantime.

**Comment by etobella** [comment]
Now it should be green

**Review by StefanRijnhart — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**